Makefile

```diff
@@ -80,20 +80,23 @@ mock-login-consent-docker:
 
 .PHONY: integration-test
 integration-test: sample-webhook-docker mock-login-consent-docker generate-test-keys
-	@cd test/integration && ENABLE_COMPOSITION=true go test -count=1 -v -cover . -p 1 -timeout=10m -race
+	@cd test/integration $$ go mod tidy && ENABLE_COMPOSITION=true go test -count=1 -v -cover . -p 1 -timeout=10m -race
 
 .PHONY: build-integration-cli
 build-integration-cli:
 	@echo "Building integration cli"
 	@mkdir -p ./build/bin
 	@cd test/integration/cli && go build -o ../../../build/bin/integration-cli main.go
 
-.PHONY: integration-test-flutter
-integration-test-flutter: build-integration-cli
+.PHONY: prepare-integration-test-flutter
+prepare-integration-test-flutter: build-integration-cli sample-webhook-docker mock-login-consent-docker generate-test-keys generate-android-bindings copy-android-bindings
 	@cd test/integration/fixtures && docker-compose -f docker-compose.yml up --force-recreate -d
-	@sleep 30
+
+.PHONY: integration-test-flutter
+integration-test-flutter:
+#	@cd test/integration/fixtures && docker-compose -f docker-compose.yml up --force-recreate -d
```

Comment:
Commented out lines? If you intended to leave them in there, could you add an explanation?